touchforms/formplayer/static/formplayer/script/webformsession.js

```diff
@@ -299,7 +299,7 @@ WebFormSession.prototype.newRepeat = function(repeat) {
 
 WebFormSession.prototype.deleteRepeat = function(repetition) {
     var juncture = getIx(repetition.parent);
-    var rep_ix = +(repetition.rel_ix().split(":").slice(-1)[0]);
+    var rep_ix = +(repetition.rel_ix().replace('_',':').split(":").slice(-1)[0]);
```

Comment:
you're sure about this guy? this is one part of the code i've been scared to touch since i didn't fully understand what the protocol was for indexing repeats